src/libaktualizr/primary/aktualizr.cc

```diff
@@ -63,9 +63,7 @@ void Aktualizr::CycleEventHandler::handle(const std::shared_ptr<event::BaseEvent
 
```

Comment:
The original plan was also to rewrite this loop to use the functions return values instead of this huge event handler. The code would then look sequential: `CheckUpdates(); Download(); Install();`.

Could be done in a separate PR if it's convenient.

Reply:
Done. How does it look to you? I still need to test it more, though.